### DISCUSSION TITLE
VIRUS

### DISCUSSION THREAD

**Discussion by TonyE2** [Dpeta/pesterchum-alt-servers#52] [Q&A] [answered]
my avast detected that has win32:malware-gen someone help me pls

**Comment by TonyE2** [comment]
oh god that was close

**Comment by Dpeta** [comment, marked as answer]
Yeah, it's nothing úvù
Builds like this sometimes get flagged because they're [unsigned](https://stackoverflow.com/questions/44377666/pyinstaller-exe-throws-windows-defender-no-publisher) and the exe is just not known to the antivirus. This can happen with both pyinstaller and [cx_freeze](https://github.com/marcelotduarte/cx_Freeze/issues/315). Builds packed with [UPX](https://stackoverflow.com/a/355581) may run into this issue more frequently as well.

Reporting it as a false positive is probably the only way to make it get flagged less frequently for other users in the future.

All ""win32:malware-gen"" means is that the files seems suspicious, but if that makes you uncomfortable you could always just install python, clone the repository, and run pesterchum.py directly. Python code doesn't need to be compiled so it's functionally the same in the end.

**Reply by TonyE2** [reply]
oh thx

**Comment by TonyE2** [comment]
ehm mmm 
![image](https://user-images.githubusercontent.com/73620588/127568656-43eeddeb-8977-406c-a378-0ba5dee3a665.png)

**Reply by Dpeta** [reply]
Hm?
You can just put an exception on the folder úvù

**Reply by TonyE2** [reply]
welp

**Reply by TonyE2** [reply]
oh thx so much

**Reply by TonyE2** [reply]
now how i use it
